ocs_ci/ocs/cluster.py

```diff
@@ -954,6 +954,28 @@ def validate_replica_data(pool_name, replica):
     raise PoolNotFound(f"Pool {pool_name} not found on cluster")
 
 
+def get_byte_used_by_pool(pool_name):
+    """
+
+    Check byte_used value for specific pool
+
+    Args:
+        pool_name (str): name of the pool to check replica
+
+    Returns:
+        integer: The amount of byte stored from pool.
+
+    """
+    ceph_df_detail_output = get_ceph_df_detail()
+    pool_list = ceph_df_detail_output.get("pools")
+    for pool in pool_list:
+        if pool.get("name") == pool_name:
+            logger.info(f"{pool_name}")
+            byte_used = pool["stats"]["bytes_used"]
+            return byte_used
+    raise PoolNotFound(f"Pool {pool_name} not found on cluster")
```

Comment:
It's better to return a value and process it at the test level so if needed, the exception will be raised there and not here

Reply:
The return value is processed at test level. This is an exception related if the pool not found at all. and the recommendation to do it this way came from Martin in another PR. Exception regarding return value is raised at the test level.

Reply:
understood now. Let's keep it as is